css/style.css

```diff
@@ -0,0 +1,185 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+main {
+  font-family: Lato, sans-serif;
+  font-size: 22px;
+}
+
+/* easy attributes */
+.dflex {
+  display: flex;
+}
+
+.background-gradient {
+  background-image: linear-gradient(rgb(114, 111, 111) 50%, rgb(97, 95, 95) 40%, rgb(97, 95, 95));
+}
+
+.z-index-top {
+  z-index: 2;
+}
+
+.z-index-bot {
+  z-index: 0;
+}
+
+.flex-ietm-1-8 {
+  flex: 1;
+  min-width: 50px;
+}
+
+.flex-item-9 {
+  flex: 1;
+}
+
+.float-l {
+  float: left;
+}
+
+.float-r {
+  float: right;
+}
+
+.centered {
+  margin: 60px auto auto auto;
+  text-align: center;
+  width: 700px;
+  color: white;
+}
+
+.centered h2 {
+  margin-bottom: 40px;
+}
+
+.white {
+  color: white;
+  text-decoration: none;
+}
+
+.link {
+  font-size: 20px;
+}
+
+.nostyle {
+  text-decoration: none;
+}
+
+.border-right {
+  border-right: 0.5px solid grey;
+  padding: none 7px none 7px;
+}
+
+.grey-text {
+  color: grey;
+}
+
+.footerpadding {
+  margin: 10px;
+  font-size: 13px;
+}
+
+.footerpadding-btm {
+  margin-bottom: 30px;
+}
+
+.margin-left {
+  margin-left: 100px;
+}
+
+.margin-rgt {
+  margin-right: 80px;
+}
+
+.link-clr {
+  color: rgb(0, 143, 209);
+}
+
+/* navbar */
+.navbar-container {
+  width: 100%;
+  height: 6em;
+  justify-self: center;
+  align-items: flex-end;
+  justify-content: center;
+  align-content: center;
+}
+
+nav {
+  height: 1.5em;  border-radius: 0px;
+  width: 79%;
+  border: 1px solid gray;
+  border-radius: 3px;
+}
+
+.nav-btn {
+  width: 100px;
+  height: 2.5em;
+  border-top: none;
+  border-bottom: none;
+  border-left: none;
+  border-right: 0.5px solid rgb(59, 58, 58);
+  color: white;
+}
+
+.nav-btn:hover {
+  background: radial-gradient(circle, rgba(88, 88, 88, 0.9995185574229691) 82%, rgba(106, 106, 106, 1) 100%);
+  cursor: pointer;
+}  border-radius: 0px;
+
+.search-icon {
+  position: absolute;
+  font-size: 15px;
+  padding-left: 2%;
+  color: white;
+}
+
+.fab {
+  font-size: 20px;
+}
+
+.searchbar-container {
+  width: 140px;
+  height: 1.48em;
+  align-items: center;
+  justify-items: center;
+}
+
+.search-query {
+  background-size: 16px;
+  justify-self: center;
+  width: 80%;
+  height: 70%;
+  border-radius: 50px;
+  margin: auto 10% auto 10%;
+}
+
+/* top background section */
+.background-1 {
+  height: 90vh;
+  max-width: 100%;
+  background: url(/pictures/bg-img.jpg) no-repeat bottom;
+}
+
+/* bottom background section */
+.background2 img {
+  width: 100%;
+  max-width: 100vh;
+  padding-top: 2px;
+}
```

Comment:
In Order to align the images well, Try and divide the container into four equal parts. This will make sure the images are aligned well regardless of the screen size